src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/mgr-summary.pipe.ts

```diff
@@ -15,18 +15,31 @@ export class MgrSummaryPipe implements PipeTransform {
     }
 
     let activeCount = this.i18n('n/a');
-    const titleText = _.isUndefined(value.active_name)
+    const activeTitleText = _.isUndefined(value.active_name)
       ? ''
       : `${this.i18n('active daemon')}: ${value.active_name}`;
-    if (titleText.length > 0) {
+    if (activeTitleText.length > 0) {
       activeCount = '1';
     }
+    let str = '';
+    let flg = false;
+    for (const i in value.standbys) {
+      if (flg) {
+        str = str + ', ' +  value.standbys[i].name;
+      } else {
+        str = str + value.standbys[i].name;
+        flg = true;
+      }
+    }
+    const standbyTitleText = !str // True for 0, Null and undefined
```

Comment:
The typescript comment in [my github comments suggestion](https://github.com/ceph/ceph/pull/33529#discussion_r384443513) was mend purely descriptive. Could you please remove the comment in this line?

Reply:
Okay